clippy_lints/src/utils/constants.rs

```diff
@@ -0,0 +1,17 @@
+//! This module contains some useful constants.
+
+#![deny(missing_docs_in_private_items)]
+
+/// List of the built-in types names.
```

Comment:
Maybe link to https://doc.rust-lang.org/reference.html#primitive-types as a source?